test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -691,8 +692,84 @@ public void GetDatabaseWithQueryConnectionTest()
 
             // remove mock from ConnectionInfo
             Assert.True(connInfo.ConnectionTypeToConnectionMap.TryRemove(ConnectionType.Query, out connection));
-        }
+        }
+
+        /// <summary>
+        /// Get Definition for a object with no definition. Expect a error result
+        /// </summary>
+        [Fact]
+        public async void GetDefinitionFromChildrenAndParents()
+        {
+            string queryString = "select * from master.sys.objects";
```

Comment:
At a minimum please add procedure-specific tests, and tests with and without escaping of brackets on each one. Ideally make it easy to define multiple inputs so future bug regressions can be tested (e.g. add in a framework that takes JSON file input defining the query & positions to test from, and what's expected to be returned at each). This 2nd ask is a stretch goal though - can ship without it.